recipe/meta.yaml

```diff
@@ -1,29 +1,26 @@
-{% set name = "pywin32" %}
-{% set version = "223" %}
+{% set version = "226" %}
 
 package:
-  name: {{ name }}
+  name: pywin32
   version: {{ version }}
 
 source:
-  url: https://github.com/mhammond/{{ name }}/archive/b{{ version }}.tar.gz
-  sha256: c8be831cb0fc11badf27b6edc0da08984322452e5c98ac84aa158cc89f6d0837
+  url: https://github.com/mhammond/pywin32/archive/b{{ version }}.tar.gz
+  sha256: 172c5618d22cf8e9bca7bc01699c4bd94d4b03c926a6da125d5fe9b8eaa2837b
 
 build:
   number: 1000
   script:
-    - "{{ PYTHON }} setup.py install --record=record.txt"  # [not py3k]
-    - "{{ PYTHON }} setup3.py install --record=record.txt"  # [py3k]
+    - "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
It seems that the separated `setup.py` were merged into one.

Reply:
👍